pom.xml

```diff
@@ -472,6 +472,23 @@
         </executions>
       </plugin>
       <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>buildnumber-maven-plugin</artifactId>
```

Comment:
@KostyaSha I prefer using things coming from MojoHaus when possible for many reasons :-). Also, I really want to put those into the MANIFEST, is this feasible with this one? After a 30 seconds look, I'm not sure it is.